src/components/BulkEditModal/BulkEditModal.test.tsx

```diff
@@ -57,15 +53,20 @@ test('successfully adds probes', async () => {
     },
   ];
 
-  const { instance, user } = renderBulkEditModal('add', checksWithASingleProbe);
+  const { record, read } = getServerRequests();
+  server.use(apiRoute(`bulkUpdateChecks`, {}, record));
```

Comment:
I'll document how to use these correctly in the dev readme once I've finished the full implemenation.